pkg/pool-utils/contracts/oracle/PoolPriceOracle.sol

```diff
@@ -77,6 +77,24 @@ abstract contract PoolPriceOracle is IPoolPriceOracle, IPriceOracle {
         return Buffer.SIZE;
     }
 
+    /**
+     * @dev Manually fill oracle slots with dummy data, so that subsequent swaps filling them with real data
+     * cost less gas. Initialize by range, since filling a slot costs roughly 20k gas, so trying to fill the
+     * entire buffer at once would run out of gas.
+     */
+    function initializeOracle(uint256 startSlot, uint256 endSlot) external {
```

Comment:
'initialize' is not a good name here: it leads people into thinking that this, well, initializes the oracle, which it doesn't.

I'd use highly-specific vocabulary here, specially considering this is a low-level utility that the vast, vast majority of people don't care about. Maybe something like `dirtyUninitializedOracleSamples`.

Reply:
initializeOracleStorage?

Reply:
No, my gripe is with the 'initialize' word: we're not initializing anything on the oracle, it is as 'empty' as it used to, there's no new data. 

What we're doing is an extremely niche and low level thing, which is dirtying the storage slot to make the next write cheaper due to it being already allocated in the node non volatile storage. The name should reflect that.